lec02-basic-r.Rmd

```diff
@@ -1,4 +1,754 @@
 ---
-title: "Basic R (part 1): Rstudio, Rmarkdown, assignment, vectors"
+title: Introduction to R
 ---
 
+**Copyright (c) Data Carpentry**
+
+*Note: This lecture content was originally created by voluntary contributions to
+[data carpentry](http://datacarpentry.org) and has been modified to
+align with the aims of EEB430. Data Carpentry is an organization focused on data
+literacy, with the objective of teaching skills to researchers to enable them to
+retrieve, view, manipulate, analyze and store their and other’s data in an open
+and reproducible way in order to extract knowledge from data. For EEB430, we are
+making all our content available under the same license, The Creative Commons, 
+license, so if someone in the future are teaching this course content available under
+the same lice*
+
+I am emphazing this because I think it is one of the core features of working
+with an open language like R, many smart people readily share their material and
+discoveries not only for free, but in a way that allows others to modify and
+built on the material themselves. It really is "standing on the shoulders of
+giants", and there is a common saying in the open source world:
+
+> I came for the language and stayed for the community.
+
+Which highlights the spirit, generousity, and the fun involved in many of these project.
+
+------------
+
+
+## Recap from intro lecture
+
+We will be working with the programming language R through an interface called
+RStudio. We will mainly be using the R Notebook, which allows us to mix R code
+with writing notes in the Markdown text format. So let's open an R Notebook in
+RStudio... You will be prompted to install a few packages, click Yes/OK. I have
+asked the IT services to make sure these packages are always installed on the
+computers so that we don't have to install them every time, hopefully they will
+do so soon. If you prefer, you can also bring your own laptop to class, which
+has the advantage that you can save things permanently and that you have
+everything setup, this is up to you <!--ok to say this?-->. Let's go over the
```

Comment:
Is it ok to offer them to bring their own computers? Any unequality that could arise from that? Should not since they all have access to computers right?